static/src/javascripts/projects/commercial/modules/messenger/resize.js

```diff
@@ -44,18 +45,28 @@ const resize = (
     return fastdom.write(() => {
         Object.assign(adSlot.style, styles);
         Object.assign(iframe.style, styles);
+        Object.assign(iframeContainer.style, styles);
     });
 };
 
 const init = (register: RegisterListeners) => {
     register('resize', (specs, ret, iframe) => {
         if (iframe && specs) {
+            /* there are ultimately three containers that need resizing:
```

Comment:
Is this comment necessary? If the containing div class name was to ever change then this comment would be invalid. I'd say your code is already self documenting given you have well named variables 👍

Reply:
Good point - I'll remove the comment.